### PR TITLE
Feature/improved engineering view

### PR DESCRIPTION
**Pull request by aluneau**
- Add a new My Queue page (/my-queue) that surfaces support cases needing engineering attention — cases where the customer has commented on the portal more recently than the engineer has on JIRA
- The view is automatically scoped to the logged-in engineer's assigned cards, so each engineer sees only their own workload
- Expandable rows show both portal and JIRA comment history side-by-side for quick context
- Add comment_type field to the Comment model with auto-migration support for schema changes
- Add a nullable no_update_date column to JiraCard and wire the read-side exclusion (mirrors the reference "no update needed" clause); stays NULL until write-back exists, so no behavior change yet
- Add 29 tests covering the core get_my_queue_cases() logic: inclusion/exclusion rules, sprint and engineer filtering, return structure, and edge cases

How it works

get_my_queue_cases() joins Cases with JIRA cards (excluding closed cases), then for each case compares the most recent portal comment timestamp against the most recent JIRA comment timestamp. A case is shown if:
- It has no JIRA comments (never responded to), or
- Its most recent portal comment is newer than its most recent JIRA comment (customer followed up)

### CODE REVIEW

File: dashboard/docker-compose.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ services:
       timeout: 5s
       retries: 5
     volumes:
-      - postgresql_data:/var/lib/postgresql/data
+      - postgresql_data:/var/lib/postgresql
```
Comment (PB1899) [inline dashboard/docker-compose.yml]: I couldn't see this in the description, was there a particular reason to change the Docker volume? 
I don't think it influences the prod deployment actually, so I'm not sure if this should be done in the PR.

**Review by PB1899 — CHANGES_REQUESTED** [review]
LGTM otherwise, see my comment about the docker-compose file.
